GeosCore/ucx_mod.F90

```diff
@@ -924,7 +921,9 @@ SUBROUTINE SETTLE_STRAT_AER( Input_Opt, State_Chm, State_Grid, State_Met, RC )
           IF (RUNCALC) THEN
              ! Need to translate for BC radii
              IF ( State_Met%InChemGrid(I,J,L) ) THEN
-                RWET(IBC) = WERADIUS(I,J,L,2)*1.e-2_fp
+                ! NOTE: Replace 2 with 2+NDUST
+                ! See: https://github.com/geoschem/geos-chem/issues/2169
```

Comment:
I would suggest removing these lines. People can always use Git blame to view the line history.

Reply:
Now removed in commit https://github.com/geoschem/geos-chem/commit/fda4d7bbfe7c16da33ad6422b9b117ee160f1a5a.